### PR TITLE
Fix classifications app unexpected scroll on safari

### LINKED ISSUE #425: Closing edit classification modal in apps/classifications causes Safari to scroll to top
`When editing incidents near the bottom of the incident list (such as incident 101) in Safari, closing out of the pop-up editing window results in jumping back up to the beginning of the incident list`

Not really clear why this happens.

### PR DESCRIPTION
**Pull request by cesarvarela**
This turned into a nightmare. 

Switches modals to Flowbite. 

More work should be done to remove the use of `useMongo,` replace it with graphql calls instead and clean up the overall architecture.

closes #425

### CODE REVIEW

File: site/gatsby-site/src/pages/apps/classifications.js [hunk 1]
```diff
@@ -311,6 +312,185 @@ const getClassificationsArray = (classifications, taxonomy) => {
...
+        ref={editFormRef}
+        namespace={taxaData.namespace}
+        incidentId={row.values.IncidentId}
+        onSubmit={onSubmit}
+      />
+    </div>
+  );
+}
+
+function Row({
```
Comment (lmcnulty) [inline site/gatsby-site/src/pages/apps/classifications.js:378]: This file is very long. Can we move some of the components into their own files?

File: site/gatsby-site/src/pages/apps/classifications.js [hunk 2]
```diff
@@ -311,6 +312,185 @@ const getClassificationsArray = (classifications, taxonomy) => {
...
+              >
+                <Modal.Header>
+                  <div className="flex items-center">
+                    <div>
+                      Edit {currentTaxonomy} classification for incident {row.original.IncidentId}
+                    </div>
+                    &nbsp;
+                    <Button className="ms-2" onClick={() => editFormRef.current.submit()}>
+                      Submit
+                    </Button>
```
[the remark this answers is not in the document]
Comment (cesarvarela) [reply site/gatsby-site/src/pages/apps/classifications.js:430]: Yes, I agree, although someone using that form extensively asked for that button to be placed there. I like, the sticky button better, feel free to open an issue about it.

File: site/gatsby-site/cypress/fixtures/call/classifications.json [hunk 3]
```diff
@@ -0,0 +1,72 @@
...
+            "Public Sector Deployment": false,
+            "Physical System": [
+                "Vehicle/mobile robot"
+            ],
+            "Data Inputs": [
+                "images of surroundings"
+            ]
+        },
+        "incident_id": {
+            "$numberInt": "97"
```
[the remark this answers is not in the document]
Comment (cesarvarela) [reply site/gatsby-site/cypress/fixtures/call/classifications.json:67]: No, that's the response when using the MongoDB js client; as I mentioned in the PR description, usage of it should be removed; it is the last place where it is being used, I believe.

(https://github.com/responsible-ai-collaborative/aiid/blob/master/site/gatsby-site/src/hooks/useMongo.js)

File: site/gatsby-site/src/pages/apps/classifications.js (hunk 1 above)
Comment (cesarvarela) [reply site/gatsby-site/src/pages/apps/classifications.js:378]: Opened this one https://github.com/responsible-ai-collaborative/aiid/issues/1563

**Comment by cesarvarela** [comment]
@lmcnulty If I didn't break anything, let's merge this one. I'll level the improvements to you since you are already working on something closely related, and you'll probably need to modify the classifications app too.

**Review by lmcnulty — APPROVED** [review]
@cesarvarela Sounds good.

**Pull request merged**